proxy/proxy/client_test.go

```diff
@@ -92,6 +92,9 @@ func TestContextDialer(t *testing.T) {
 	if _, err := c.DialContext(context.Background(), instance); err != errFakeDial {
 		t.Errorf("unexpected error: %v", err)
 	}
+
+	// leakcheck.RegisterIgnoreGoroutine("go.opencensus.io@v0.22.5/stats/view/worker.go")
+	// leakcheck.Check(t)
```

Comment:
Leaving this here for reviewers to see how I wrote a failing test with the leak, and then fixed it. Before merging this PR, these two lines will be deleted.